### DISCUSSION TITLE
bottlecrm.com doesn't load

### DISCUSSION THREAD

**Discussion by lcfd** [Django-CRM/Django-CRM#488] [General]
Hi, I'm trying to use [bottlecrm.com](https://bottlecrm.com/) but it doesn't load.
Is everything ok?

**Comment by codecakes** [comment]
it's bottlecrm.io now

**Reply by rafalkrupinski** [reply]
still doesn't load

**Comment by i7solutions** [comment]
bottle.com.io is also not working
